bigbluebutton-html5/imports/ui/components/external-video-player/external-video-player-graphql/component.tsx

```diff
@@ -277,6 +311,12 @@ const ExternalVideoPlayer: React.FC<ExternalVideoPlayerProps> = ({
     return 1;
   }, []);
 
+  useEffect(() => {
+    if (playerUrl !== videoUrl) {
+      setPlayerUrl(addTimeParamToTwitchUrl(videoUrl, getCurrentTime()));
```

Comment:
Will everyone run it or just the presenter?
I think it's necessary only for the presenter, right?